webpack/components/AnsibleHostDetail/AnsibleHostDetail.test.js

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
-
+import { STATUS } from 'foremanReact/constants';
+import { testComponentSnapshotsWithFixtures } from 'foremanReact/common/testHelpers';
 import AnsibleHostDetail from './';
 
+const fixtures = {
+  'should render Ansible Host Detail': {
+    response: {},
+    status: STATUS.RESOLVED,
+  },
+};
+
 describe('AnsibleHostDetail', () => {
-  it('should show content', () => {
-    render(<AnsibleHostDetail />);
-    expect(
-      screen.getByText('Ansible Variables coming soon!')
-    ).toBeInTheDocument();
-  });
+  describe('rendering', () =>
+    testComponentSnapshotsWithFixtures(AnsibleHostDetail, fixtures));
```

Comment:
I know the existing test was complaining about component not being wrapped in Provider and the snapshot tests are seductively easy to write using those test helpers, but if you take a look at the generated snapshot, it really does not test much and it is not able to detect what is on the page.

I would stay away from using snapshots and modified the existing test so that is passes, let me know if you need a hint.